autoload/codeclimate.vim

```diff
@@ -16,7 +16,7 @@ if !exists('g:vimcodeclimate_analyze_cmd')
   let g:vimcodeclimate_analyze_cmd = 'codeclimate analyze'
 endif
 
-function! s:AnalyzeProject()
+function! s:AnalyzeProject() abort
```

Comment:
¯\\\_(ツ)\_/¯. Vint told me to, and references [Google's styleguide](https://google.github.io/styleguide/vimscriptguide.xml?showone=Functions#Functions).

I think it's effectively to prevent a bug in one function from bringing down everything?